pkg/graph/schema.resolvers.go

```diff
@@ -116,6 +116,126 @@ func (r *accessibilityRequestDocumentResolver) UploadedAt(ctx context.Context, o
 	return obj.CreatedAt, nil
 }
 
+func (r *businessCaseResolver) AlternativeASolution(ctx context.Context, obj *models.BusinessCase) (*model.BusinessCaseSolution, error) {
+	return &model.BusinessCaseSolution{
+		AcquisitionApproach:     obj.AlternativeAAcquisitionApproach.Ptr(),
+		Cons:                    obj.AlternativeACons.Ptr(),
+		CostSavings:             obj.AlternativeACostSavings.Ptr(),
+		HasUI:                   obj.AlternativeAHasUI.Ptr(),
+		HostingCloudServiceType: obj.AlternativeAHostingCloudServiceType.Ptr(),
+		HostingLocation:         obj.AlternativeAHostingLocation.Ptr(),
+		HostingType:             obj.AlternativeAHostingType.Ptr(),
+		Pros:                    obj.AlternativeAPros.Ptr(),
+		SecurityIsApproved:      obj.AlternativeASecurityIsApproved.Ptr(),
+		SecurityIsBeingReviewed: obj.AlternativeASecurityIsBeingReviewed.Ptr(),
+		Summary:                 obj.AlternativeASummary.Ptr(),
+		Title:                   obj.AlternativeATitle.Ptr(),
+	}, nil
+}
+
+func (r *businessCaseResolver) AlternativeBSolution(ctx context.Context, obj *models.BusinessCase) (*model.BusinessCaseSolution, error) {
+	return &model.BusinessCaseSolution{
+		AcquisitionApproach:     obj.AlternativeBAcquisitionApproach.Ptr(),
+		Cons:                    obj.AlternativeBCons.Ptr(),
+		CostSavings:             obj.AlternativeBCostSavings.Ptr(),
+		HasUI:                   obj.AlternativeBHasUI.Ptr(),
+		HostingCloudServiceType: obj.AlternativeBHostingCloudServiceType.Ptr(),
+		HostingLocation:         obj.AlternativeBHostingLocation.Ptr(),
+		HostingType:             obj.AlternativeBHostingType.Ptr(),
+		Pros:                    obj.AlternativeBPros.Ptr(),
+		SecurityIsApproved:      obj.AlternativeBSecurityIsApproved.Ptr(),
+		SecurityIsBeingReviewed: obj.AlternativeBSecurityIsBeingReviewed.Ptr(),
+		Summary:                 obj.AlternativeBSummary.Ptr(),
+		Title:                   obj.AlternativeBTitle.Ptr(),
+	}, nil
+}
+
+func (r *businessCaseResolver) AsIsSolution(ctx context.Context, obj *models.BusinessCase) (*model.BusinessCaseSolution, error) {
+	return &model.BusinessCaseSolution{
```

Comment:
This solution only uses a subset of the available fields. Should we make a separate type for it?

Reply:
Yes, I think we should